tests/rules/property.js

```diff
@@ -145,7 +147,47 @@ eslintTester.run("property", rule, {
             ]
         },
 
-
+        // Typescript support valid cases
+        // raw strings can be assigned to innerHTML
+        {
+            code: "(options as HTMLElement).innerHTML = '<s>safe</s>';",
+            parser: PATH_TO_TYPESCRIPT_ESLINT,
+            parserOptions: {
+                ecmaVersion: 2018,
+                sourceType: "module",
+            }
+        },
+        {
+            code: "(<HTMLElement>items[i](args)).innerHTML = 'rawstring';",
+            parser: PATH_TO_TYPESCRIPT_ESLINT,
+            parserOptions: {
+                ecmaVersion: 2018,
+                sourceType: "module",
+            }
+        },
+        {
+            code: "lol.innerHTML = (5 as string);",
+            parser: PATH_TO_TYPESCRIPT_ESLINT,
+            parserOptions: {
+                ecmaVersion: 2018,
+                sourceType: "module",
+            },
+        },
+        {
+            code: "node!.innerHTML = DOMPurify.sanitize('<em>${evil}</em>');",
```

Comment:
That test case is borked, because the supplied string is not a template string.
For simplicity's sake, do `.sanitize(evil)` instead, please.

Reply:
done